docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -655,6 +655,63 @@ the same Kubernetes cluster or installing the CRDs on your own you should not ha
 
 ## `operator`
 
+### `operator.annotations.deployment`
+
+| Type     | Default value |
+|----------|---------------|
+| `object` | `{}`          |
+
+[Kubernetes reference](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+
+Kubernetes annotations which should be applied to the `Deployment` created by the chart.
+
+`values.yaml` example:
+
+  ```yaml
+    operator:
```

Comment:
Can you reduce the identation here?

Reply:
Hello @tigrato 
Thanks for your feedback. Is that better now ?
Thank you

Reply:
It's perfect.